doc/source/examples/permutation_importance_classification_leave.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>Is it worth mentioning the change of scale for the importance scores as compared to the previous example using a metric? I.e. here we have scores that are generally an order of magnitude higher than previously, although the only change is replacing a metric with a <code>1-metric</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>